tests/components/ct-vue-vite/src/router/index.ts

```diff
@@ -0,0 +1,11 @@
+import DashboardPage from '../pages/DashboardPage.vue';
+import LoginPage from '../pages/LoginPage.vue';
+import { createRouter, createWebHistory } from 'vue-router';
+
+export const router = createRouter({
+  history: createWebHistory('playwright/index.html'),
```

Comment:
I can't get rid of `/playwright/index.html` until https://github.com/microsoft/playwright/pull/17767 is fixed